projects/gnoland/gno.land/r/metamodel000/diagram.gno

```diff
@@ -0,0 +1,35 @@
+package metamodel
+
+import (
+	mm "gno.land/p/metamodel000"
+)
+
+var docReviewDescription = `
+This model replicates [metamodel000:coffee-shop](metamodel000:coffee-shop) using the ImportDiagram feature.
+
+`
+
+func init() {
+	diagram := `
+$Water --> BoilWater
+BoilWater --> $BoiledWater
+$CoffeeBeans --> GrindBeans
+GrindBeans --> $GroundCoffee
+$BoiledWater --> BrewCoffee
+$GroundCoffee --> BrewCoffee
+$Filter --> BrewCoffee
+BrewCoffee --> $CoffeeInPot
+$CoffeeInPot --> PourCoffee
+$Cup --> PourCoffee
+$Pending --> Send
+Send --> $Sent
+$Sent --> Credit
+Credit --> $Payment
+PourCoffee -|> $Payment
+ `
+	model := mm.ImportDiagram(diagram)
```

Comment:
here's the Ascii syntax - no weights supported 

I think this is idea for building workflows, or defining arrow-based relationships

Reply:
one of the assumptions here is that if there is no input to a given state it's assumed to be pre-populated at the start

i.e. pre-reqs like ingredients must exist - all other places are initialized to 0